pkg/blob/controllerserver.go

```diff
@@ -114,7 +114,9 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	}
 	if IsAzureStackCloud(d.cloud) {
 		accountKind = string(storage.Storage)
-		storageAccountType = "Standard_LRS"
+		if storageAccountType != string(storage.StandardLRS) && storageAccountType != string(storage.PremiumLRS) {
+			return nil, status.Errorf(codes.InvalidArgument, fmt.Sprintf("Invalid skuName value: %s, as Azure Stack only supports %s and %s Storage Account types.", storageAccountType, storage.PremiumLRS, storage.StandardLRS))
+		}
```

Comment:
@andyzhangx I took your advice to throw errors when unsupported sku is requested. Accordingly, I changed the test logic a bit. Please let me know your thoughts. If this is OK, I will make a similar change against Azure Disk CSI.